server/services/withings/lib/utils/withings.buildBatteryLevelValues.js

```diff
@@ -17,11 +17,11 @@ function buildBatteryLevelValues(currentBatValueString) {
       break;
     case 'medium':
       batValueString = `${currentBatValueString} (> 30%)`;
```

Comment:
You changed the batValue from 30 to 50, but not the text!

Reply:
i fix this